### PR TITLE
Parsejobdetail

### PR DESCRIPTION
**Pull request by tiantn**
Updated parseJobDetails part.
Fixed can't list job with owner * problem.
Updated list job unit test and system test.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the z/OS FTP Plug-in for Zowe CLI will be documented in this file.
 
+## Recent Changes
+
+- Fixed list jobs problems.
+- Updated list jobs unit test and system test.
```
Comment (zFernand0) [inline CHANGELOG.md:8]: It doesn't hurt to have the extra info about the test being updated, however we usually leave out anything that may not impact end users.

**Review by zFernand0 — APPROVED** [review]
LGTM!
Thanks for updating the tests! : )

File: CHANGELOG.md (hunk 1 above)
Comment (tiantn) [reply CHANGELOG.md:8]: Thank you @zFernand0! I got it. 😊

**Review by lauren-li — APPROVED** [review]
This PR looks good to me! All unit and expected system tests are passing, and the zFTP CLI command to list jobs with the `--owner "*"` option is working well in manual testing.

Thanks @tiantn for this bugfix!

**Pull request merged**